genai-perf/README.md

```diff
@@ -89,7 +89,7 @@ genai-perf --help
 Since GenAI-Perf depends on Perf Analyzer,
 you'll need to install the Perf Analyzer binary:
 
-### Install Perf Analyzer (Ubuntu, Python 3.8+)
+### Install Perf Analyzer (Ubuntu, Python 3.10+)
```

Comment:
@dyastremsky I think this change was missing its README_template associated change.

I fixed it in https://github.com/triton-inference-server/perf_analyzer/pull/150

Reply:
Awesome, thank you! It's been a bit of extra effort to keep the docs + doc templates in sync.